### PR TITLE
Build/clean v2 project form

### PR DESCRIPTION
**Pull request by andre-code**
This PR is about making improvements to the v2 new project form.

### Changes:
- After successfully creating a new project, the form will be cleaned up.
- When a new project is created, the project list will be refreshed automatically.
- We're adding better btn styles, spacing, and icons to make the new project form easier to use.

/deploy #notest

### CODE REVIEW

**Review by ciyer — COMMENTED** [review]
Very nice! 🚀 

There was a weird error in the CI pipeline. I am re-running, I will approve once CI passes.

File: client/src/features/projectsV2/new/ProjectV2New.tsx [hunk 1]
```diff
@@ -202,6 +205,10 @@ function ProjectV2NewReviewCreateStep({
 
 export default function ProjectV2New() {
   const user = useLegacySelector((state) => state.stateModel.user);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(setCurrentStep(0));
+  }, []); // eslint-disable-line
```
Comment (leafty) [inline client/src/features/projectsV2/new/ProjectV2New.tsx:211]: Please do no ignore the rules of hooks.

**Review by leafty — CHANGES_REQUESTED** [review]
One comment about a hook, otherwise seems good.

File: client/src/features/projectsV2/new/ProjectV2New.tsx (hunk 1 above)
Comment (andre-code) [reply client/src/features/projectsV2/new/ProjectV2New.tsx]: added in the last commit

**Review by leafty — APPROVED** [review]

**Pull request merged**